backend/src/polling_station/structs.rs

```diff
@@ -33,10 +61,10 @@ pub enum PollingStationType {
 impl From<String> for PollingStationType {
     fn from(value: String) -> Self {
         match value.as_str() {
-            "vaste_locatie" => Self::FixedLocation,
-            "bijzonder" => Self::Special,
-            "mobiel" => Self::Mobile,
-            _ => panic!("invalid PollingStationType"),
+            "FixedLocation" => Self::FixedLocation,
```

Comment:
Maybe to a case insensitive match here?

(untested)
```rust
impl From<String> for PollingStationType {
    fn from(value: String) -> Self {
        match value.as_str().to_lowercase().as_str() {
            "fixedlocation" => Self::FixedLocation,
            "special" => Self::Special,
            "mobile" => Self::Mobile,
            _ => panic!("invalid PollingStationType `{value}`"),
        }
    }
}
§§§

Reply:
Don't we just want to fix the typo in the fixture?

Reply:
I +1 this option

Reply:
Fixed in [9aa737d](https://github.com/kiesraad/abacus/commit/9aa737d1033219f23e209d27efc1f33ace78ee46)

Reply:
My bad - as I understand from other Marlon this is a enum we fully control ourselves - don't apply my suggestion :-)